### PR TITLE
doc: release-notes-latest: Add changes in drivers from upmerge 20201214

### PR DESCRIPTION
**Pull request by anangl**
Note changes related to Zephyr drivers that are relevant to NCS.

Signed-off-by: Andrzej Głąbek <andrzej.glabek@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/releases/release-notes-latest.rst [hunk 1]
```diff
@@ -385,20 +387,40 @@ The following list summarizes the most important changes inherited from upstream
     * Fixed a regression in lazy loading of the Client Configuration Characteristics.
     * Fixed an issue where a security procedure failure could terminate the current GATT transaction when the transaction did not require security.
 
+  * Clock control:
+
+    * Changed definition (parameters and return values) of API function :c:func:`clock_control_async_on`.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:392]:
```suggestion
    * Changed the definition (parameters and return values) of the API function :c:func:`clock_control_async_on`.
```

File: doc/nrf/releases/release-notes-latest.rst [hunk 2]
```diff
@@ -385,20 +387,40 @@ The following list summarizes the most important changes inherited from upstream
...
+    * Added support for the audio clock in nRF53 Series SoCs.
+    * Added missing handling of the HFCLK192M_STARTED event in nRF53 Series SoCs.
+
+  * Counter:
+
+    * Excluded selection of nRF TIMER0 and RTC0 when Bluetooth controller is enabled.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:398]:
```suggestion
    * Excluded selection of nRF TIMER0 and RTC0 when the Bluetooth Controller is enabled.
```

File: doc/nrf/releases/release-notes-latest.rst [hunk 3]
```diff
@@ -409,6 +431,9 @@ The following list summarizes the most important changes inherited from upstream
     * Reworked the command handler reading routine, to prevent data loss and reduce RAM usage.
     * Added the possibility of locking TX in the command handler.
     * Improved handling of HW flow control on the RX side of the UART interface.
+    * Added the possibility of defining commands with variable number of arguments.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:434]:
```suggestion
    * Added the possibility of defining commands with a variable number of arguments.
```

File: doc/nrf/releases/release-notes-latest.rst [hunk 4]
```diff
@@ -409,6 +431,9 @@ The following list summarizes the most important changes inherited from upstream
...
+    * Introduced :c:func:`gsm_ppp_start` and :c:func:`gsm_ppp_stop` functions, to allow restarting the networking stack without rebooting the device.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:435]:
```suggestion
    * Introduced :c:func:`gsm_ppp_start` and :c:func:`gsm_ppp_stop` functions to allow restarting the networking stack without rebooting the device.
```

File: doc/nrf/releases/release-notes-latest.rst [hunk 5]
```diff
@@ -443,10 +469,16 @@ The following list summarizes the most important changes inherited from upstream
       * ``LINE_CTRL_DCD``
       * ``LINE_CTRL_DSR``
 
+    * Refactored :c:func:`uart_poll_out` implementation in the nRF UARTE driver to fix incorrect handling of HW flow control and power management.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:472]:
```suggestion
    * Refactored the :c:func:`uart_poll_out` implementation in the nRF UARTE driver to fix incorrect handling of HW flow control and power management.
```

File: doc/nrf/releases/release-notes-latest.rst [hunk 6]
```diff
@@ -443,10 +469,16 @@ The following list summarizes the most important changes inherited from upstream
...
+
   * SPI:
 
     * Added support for SPI emulators.
 
+  * Timer:
+
+    * Extended the nRF RTC Timer driver with vendor-specific API that allows use of remaining compare channels of the RTC that provides the system clock.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:480]:
```suggestion
    * Extended the nRF RTC Timer driver with vendor-specific API that allows using the remaining compare channels of the RTC that provides the system clock.
```

**Review by ru-fu — CHANGES_REQUESTED** [review]
Just some very small language updates. :-)

**Review by ru-fu — APPROVED** [review]
Looks good, thanks!

**Review by carlescufi — APPROVED** [review]

**Comment by carlescufi** [comment]
Merging since this is doc-only and the doc build succeeded.

**Pull request merged**